### PR TITLE
Doc 9499 document how to use an external prometheus to consume couchbase stats

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

File: modules/manage/pages/monitor/monitor-intro.adoc [hunk 1]
```diff
@@ -49,4 +49,6 @@ Statistics for the Index Service can be managed by means of Couchbase Web Consol
 
 The monitoring of statistics related to the Query Service is described in xref:manage:monitor/monitoring-n1ql-query.adoc[Monitor Queries].
 
+It's also possible to set up a Prometheus monitor to consume metrics data from a Couchbase cluster. You can find an introduction on how to do this in xref:monitor/set-up-prometheus-for-monitoring.adoc[]
```
Comment (shivaniguptacb) [inline modules/manage/pages/monitor/monitor-intro.adoc]: I suggest making a separate subsection on this page called 'Monitoring with External Prometheus' and putting these 2 sentences under that subsection.

**Review by shivaniguptacb — APPROVED** [review]
Looks good other than the 1 change I suggested.

File: modules/manage/pages/monitor/monitor-intro.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/manage/pages/monitor/monitor-intro.adoc]: Done

File: modules/manage/pages/monitor/monitor-intro.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ Statistics are retrieved based on the specification of one or more _metrics_.
 Optionally, the statistics can be further defined through the specifying of a _function_; and/or _labels_ with values.
 An instance of _Prometheus_ runs on each node of the cluster; and the metrics for each node are duly stored in that node’s instance of Prometheus.
 
-For a complete lists of metrics, see the xref:metrics-reference/metrics-reference.adoc[Metrics Reference].
+For a complete lists of metrics, see the xref:metrics-reference:metrics-reference.adoc[Metrics Reference].
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/monitor-intro.adoc]:
```suggestion
For a complete list of metrics, see the xref:metrics-reference:metrics-reference.adoc[Metrics Reference].
```

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+.couchbase_sd_config_my-cluster.yml
+[source, yaml]
+----
+- targets:
+    - '10.144.220.101:8091'
+    - '10.144.220.102:8091'
+----
+
+Now create a new directory on your system and copy the file to new directory:
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc]:
```suggestion
Now, create a new directory on your system and copy the file to the new directory:
```

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc [hunk 4]
```diff
@@ -0,0 +1,100 @@
+= Using Your Own Prometheus Server to Consume Couchbase Metrics
+:description: With Couchbase 7.x, it's not necessary to write a custom Prometheus Exporter.This page will demonstrate how to enable a Prometheus server to read from Couchbase's Prometheus-ready endpoints.
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc]: Wonder if this is a bit long for the SEO description, maybe shorten this and add more in the abstract?

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+[abstract]
+{description}
+
+== Prerequisites
+
+In this tutorial, we're going to use a https://www.docker.com/get-started[Docker] instance of https://prometheus.io/[Prometheus] to build our consumer. Make sure that you have Docker installed and running on your machine. You should also have an available Couchbase cluster with at least one running node.
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc]: For ease of reviewing we tend to put new sentences on a newline.
https://docs.couchbase.com/home/contribute/pages.html#ventilate

There might be a few other sections where this needs to be updated, but as an example:

```suggestion
In this tutorial, we're going to use a https://www.docker.com/get-started[Docker] instance of https://prometheus.io/[Prometheus] to build our consumer.
Make sure that you have Docker installed and running on your machine.
You should also have an available Couchbase cluster with at least one running node.
```

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+
+:sectnums:
+
+== Prepare the target configuration file
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc:18]: Missing blank line after title.

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+----
+<.> The port through which the Prometheus server is accessed and takes the form: `-p{nbsp}host-port:container-port`.
++
+[horizontal]
+*host-port*:: the port of the host machine used to access the Prometheus service.
+*container-port*:: this is the port used internally by the container. The host port maps on to this.
+
+<.> This is the mapping of the Prometheus configuration directory: the host machine configuration that you just created `~/prometheus` will be mapped on to the Prometheus instance configuration directory inside the Docker container.
+
+== Test your configuration.
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc:102]: Missing blank line after title.

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+Once the Docker container has started up, visit `http://localhost:9090/targets` to ensure that Prometheus is accessing your cluster correctly:
+
+image::monitor/prometheus-alert.png[]
+
+
+
```
Comment (maria-robobug) [inline modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc]: Quite a few extra blank lines here, do we need these?

**Review by maria-robobug — APPROVED** [review]
LGTM, just added some minor comments 👍 .

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc (hunk 5 above)
Comment (RichardSmedley) [reply modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc]: Ease of reviewing, yes - in particular with diffs when a page is updated later.
(have fixed in my gardening edit).

**Review by RichardSmedley — APPROVED** [review]

File: modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc (hunk 4 above)
Comment (RichardSmedley) [reply modules/manage/pages/monitor/set-up-prometheus-for-monitoring.adoc]: Effectively, only the first couple of dozen words in the description get used by Google.

**Review by simon-dew — COMMENTED** [review]
LGTM, just wondering why a tutorial? Most of the Manage section is how-to guides.

**Comment by simon-dew** [comment]
> Most of the Manage section is how-to guides

I mean, the information type is not very clearly defined in this section, but I would guess it's closer to how-to guide than anything else. Maybe information typing is something we need to start paying more attention to?

**Comment by RayOffiah** [comment]
Mmm. 🤔

I think you have a point there. It's not really a springboard to further learning (tutorial). It's basically telling you how to do something. I'll have a think about reworking it. --> @simon-dew

**Pull request merged**